spec/factories/work.rb

```diff
@@ -5,54 +5,72 @@
     factory :shakespeare_and_company_work do
       title { "Shakespeare and Company Project Dataset: Lending Library Members, Books, Events" }
       collection { Collection.research_data }
+      doi { "https://doi.org/10.34770/pe9w-x904" }
+      ark { "ark:/88435/dsp01zc77st047" }
```

Comment:
All the `Work` factories now have these two fields. Notice that I deleted the `Dataset` factories that were the one to one mapping to these Works.